src/schema/nmdc.yaml

```diff
@@ -2118,8 +2198,16 @@ enums:
       illumina:
         aliases:
           - Illumina
-        exact_mappings:
-          - BAO:0200000
+        meaning: BAO:0200000
```

Comment:
why are you using BAO:0200000 and not OBI:0000759 to represent illumina?

we should be suspicious of using new terms that require adding a new prefix expansion.

Reply:
@turbomam  we had a conversation about this during the Berkeley refactor. The OBI term was imported where as the BAO term wasn't so I thought the guidance at the time was to use BAO. Let me know if you want me to change this back.

Reply:
updated to OBI term